hack/what-to-cherry-pick.sh

```diff
@@ -4,7 +4,7 @@ set -eu
 
 br=$1;# branch
 commitPrefix=$2;# examples: fix, chore(deps), build, ci
-commitGrepPattern="^${commitPrefix}:.*(#"
+commitGrepPattern="^${commitPrefix}(*.*)*:.*(#"
```

Comment:
I see we're using [this](https://github.com/amannn/action-semantic-pull-request) to check the name, where it says:
`Valid pull request titles:

fix: Correct typo
feat: Add support for Node.js 18
refactor!: Drop support for Node.js 12
feat(ui): Add Button component`
Should we be handling that exclamation mark too?

Reply:
Exclamation marks indicate that those commits are breaking which we don’t want to cherry-pick into patch/minor releases.

Reply:
ahh, okay

Reply:
I suppose another option would be:
`commitGrepPattern="^${commitPrefix}(?.*)?:.*(#"` which would be just zero or one parentheses, but I'm good with this too.